src/components/PaymentInfoLine.vue

```diff
@@ -63,14 +146,20 @@ function fiatAmountInfoValidator(value: any) {
         && typeof value.currency === 'string';
 }
 
-@Component({components: {Account, Timer, Amount, FiatAmount, ArrowRightSmallIcon}})
+@Component({components: {Account, Timer, Amount, FiatAmount, Tooltip, ArrowRightSmallIcon}})
 class PaymentInfoLine extends Vue {
-    private get originDomain() {
-        return this.origin.split('://')[1];
-    }
+    private static readonly REFERENCE_RATE_UPDATE_INTERVAL = 60000; // every minute
+    private static readonly RATE_DEVIATION_THRESHOLD = .1;
 
     @Prop({type: Object, required: true, validator: cryptoAmountInfoValidator}) public cryptoAmount!: CryptoAmountInfo;
     @Prop({type: Object, validator: fiatAmountInfoValidator}) public fiatAmount?: FiatAmountInfo;
+    // Note that vendorMarkup and networkFee have no effect if fiatAmount is not set, as the tooltip in which they
+    // appear is only visible when fiatAmount is set. As the fiatAmount was only introduced in the v2 checkout request
+    // in the Hub, the tooltip and vendorMarkup and networkFee are thus never visible for v1 checkout requests. This
+    // should be ok though as the vendorMarkup also only exists in v2 and the free-service-info doesn't make too much
```

Comment:
I still think "This should be ok though" should not be put into a comment in the sourc code.